documentation/docs/tutorials/tutorial-1a/tutorial-1a.md

```diff
@@ -0,0 +1,75 @@
+---
+description: This tutorial will guide you through the process of setting up and conducting an experiment for causal classification modeling using H2O LLM Studio. It covers how to import datasets, configure key experiment settings, and create a new experiment.
+---
+
+import Icon from "@material-ui/core/Icon";
+
+# Create an experiment for causal classification modeling
+
+## Overview
+
+This tutorial will guide you through the process of setting up and conducting an experiment for causal classification modeling using H2O LLM Studio. It covers how to import datasets, configure key experiment settings, and create a new experiment. By following these steps, you will learn how to design experiments that can identify causal relationships in classification tasks.
+
+## Objectives
+
+1. Learn how to import datasets from Hugging Face into H2O LLM Studio.
+2. Set up an experiment for causal classification modeling with appropriate parameters.
+
+## Prerequisites
+
+1. Access to the latest version of H2O LLM Studio.
+2. Basic understanding of classification and causal models.
+
+## Step 1: Import dataset
+
+For this tutorial, we'll use the IMDb movie review dataset on Hugging Face. The dataset contains 25,000 movie reviews for training, each labeled as either positive or negative. Let's import the dataset.
+
+1. Click on **Import dataset**.
+2. Select **Hugging Face** as the data source from the **Source** dropdown. 
+3. In the **Hugging Face dataset** field, enter `stanfordnlp/imdb`.
+4. In the **Split** field, enter `train`.
+5. Click **Continue**.
+
+![Import dataset](import-dataset.png)
+
+## Step 2: Configure dataset
+
+In this step, we'll review and adjust the dataset settings for our experiment.
+
+1. In the **Dataset name** field, enter `classification`.
+2. In the **Problem type** dropdown, select **Causal classification modeling**.
```

Comment:
Shall we mention here that if the dataset was set up correctly, the 'causal classification modeling' problem type would be selected already?  It can serve as verification to the user that the dataset was set up correctly.